content/work/bremtown.md

```diff
@@ -102,3 +103,100 @@ all available product options.
 {% endcall %}
 
 {% endcall %}
+
+## Brand & Logo
+
+We started with brand and logo redesign
+to establish the new look and feel
+that would reflect
+Bremtown's reimagination
+of the custom cabinetry industry.
+
+## Responsive & Accessible Website
+
+Working within the structure
+of the new brand
+as well as some established layouts and components,
+we designed a responsive website
+that meets WCAG 2.1 accessibility standards
+with CMS components
+that are user-friendly and scalable.
+
+<p style="font-size: 1.5rem;">
+Bremtown's website is responsive
+and meets WCAG accessibility standards.
+<p>
+
+We helped craft the marketing messages
+and recommended images
+to support the
+cabintry-reimagined narrative
+and to reach
+three different types
+of website visitors
+with different priorities.
+
+{% call embed.media_block(
+  media=embed.img(
+    src='work/bremtown/marketing-spec-site-responsive.png',
+    alt='Bremtown marketing site for designers to browse cabinetry products on large and small screens',
+    caption='Rebranded, accessible, and responsive marketing website with imagined marketing narrative'
+  ),
+  name='desktop-work'
+) %}
+{% endcall %}
+
+## Web Platform
+
+We created a brand new web platform
+based on Bremtown's printed spec book
+for marketing their products
+to dealers, interior designers,
+and design firms
+with the goal of establishing
+new partnerships more quickly
+and streamlining existing processes
+for current partners.
+The interactive User's Guide
+teaches new customers
+how to use the spec site
+with an intuitive, self-guided process.
+
+<p style="font-size: 1.5rem;">
+Within weeks of launch,
+Bremtown reported an increase
+in the rate of new partnerships!
+<p>
+
+Starting with wireframes,
+we iterated quickly
+based on end-user feedback.
+Next we created
+interactive mockups
+showing user flow
+and component interaction,
+creating a visual style guide
+of components along the way
```

Comment:
did we do this? I don't think there's anything we could point to if someone asked to see it. I guess we have a button component.

There is a bit of a style guide for the marketing website components. I think this section is about the spec site? Maybe a separate heading above this paragraph talking about that workflow and the shared styles between the site and the app would tie this together and highlight what is unique to this project?